### PR TITLE
In reports, display PVG values rather than codes.

### PR DESCRIPTION
**Pull request by ppettitau**

### CODE REVIEW

File: rdrf/report/schema.py [hunk 1]
```diff
@@ -104,6 +104,12 @@ def cde_resolver(cdes, _info, cde_model):
                     value = cde_value["value"]
                     if datatype == 'lookup':
                         return get_widget_class(cde_model.widget_name).denormalized_value(value)
+                    elif cde_model.pv_group:
+                        cde_values_dict = cde_model.pv_group.cde_values_dict
```
Comment (ScriptSmith) [inline rdrf/report/schema.py:108]: Possible to prefetch / cache the pv_group?

File: rdrf/report/schema.py [hunk 2]
```diff
@@ -104,6 +104,12 @@ def cde_resolver(cdes, _info, cde_model):
                     value = cde_value["value"]
                     if datatype == 'lookup':
                         return get_widget_class(cde_model.widget_name).denormalized_value(value)
+                    elif cde_model.pv_group:
```
Comment (ScriptSmith) [inline rdrf/report/schema.py:107]: So nobody would find using the codes valuable in some circumstances?

**Review by ScriptSmith — CHANGES_REQUESTED** [review]

File: rdrf/report/schema.py (hunk 1 above)
Comment (ppettitau) [reply rdrf/report/schema.py:108]: The parse_utils.prefetch_form_data method (where the form, section, cdes for reporting are sourced from), is cached and also preselects the pv_group. 
https://github.com/eresearchqut/trrf/blob/next_release/rdrf/rdrf/forms/dsl/parse_utils.py#L53

That should cover it?

File: rdrf/report/schema.py (hunk 2 above)
Comment (ppettitau) [reply rdrf/report/schema.py:107]: Potentially. I could add a field to the report designer to capture which one to use. But should I check first with Yvette & Megan if there is a use case for it atm?

Comment (ScriptSmith) [reply rdrf/report/schema.py:107]: Yes please, just double check. From discussion today, it seemed like neither did.

Comment (ppettitau) [reply rdrf/report/schema.py:107]: @ScriptSmith , I have checked with Yvette and Megan and neither could think of a valid use case for requiring the PVG codes.

**Review by ScriptSmith — APPROVED** [review]

**Pull request merged**